api-inference-community/docker_images/stanza/tests/test_api_token_classification.py

```diff
@@ -59,10 +59,16 @@ def test_simple(self):
         content = json.loads(response.content)
         self.assertEqual(type(content), list)
 
-        self.assertEqual(
-            set(k for el in content for k in el.keys()),
-            {"entity_group", "word", "start", "end", "score"},
-        )
+        if "entity_group" in (el.keys() for el in content):
+            self.assertEqual(
+                set(k for el in content for k in el.keys()),
+                {"entity_group", "word", "start", "end", "score"},
+            )
+        elif "pos_group" in (el.keys() for el in content):
+            self.assertEqual(
+                set(k for el in content for k in el.keys()),
+                {"pos_group", "word", "start", "end", "score"},
+            )
```

Comment:
Quick suggestions
* Try avoiding duplicating too much content if possible
* We still have to check that one of `pos_group` and `entity_group` is present in the keys. 


Something like this, but maybe there are better ideas
```suggestion
        keys = set(el.keys() for el in content)
        self.assertTrue(any((True for x in keys if x in ["pos_group", "entity_group"])))
        key = "entity_group" if "entity_group" in keys else "pos_group"
        self.assertEqual(
              set(k for el in content for k in el.keys()),
              {key, "word", "start", "end", "score"},
          )
```

Reply:
I got rid of it altogether, I'll see if it works